README.md

```diff
@@ -50,24 +122,28 @@ Install the added dependencies
 pip install -e ".[dev]"
 ```
 
-## Publishing the package
+### Step 7: Publish Your Package
 
-Update the metadata under `project` in the pyproject.toml file to include name, description, author-name, author-email and version.
-Also, update the URL for the repository under `project.urls`.
+Once your project is implemented, you may publish it as subpackage of AutoRA. No worries though, in case you are 
```

Comment:
```suggestion
Once your project is implemented, you may publish it as subpackage of AutoRA. If you have not thoroughly vetted your project or would otherwise like to refine it further, you may 
```